### DISCUSSION TITLE
Turbopack issue with "questionable" third-party CSS

### DISCUSSION THREAD

**Discussion by jasontaylor137** [vercel/next.js#52612] [App Router]
I almost have Turbopack working on my next.js app.  Pages that don't use third-party CSS seem to be doing well.  However, a page that uses a component that forces me to include external CSS results in errors.  The component is react-odometerjs, and a sample of the offending CSS is:

.odometer.odometer-auto-theme, .odometer.odometer-theme-default {
  display: inline-block;
  vertical-align: middle;
  *vertical-align: auto;
  *zoom: 1;
  *display: inline;
  position: relative;
}

The lines with properties starting with "*" are the ones that fail.  I can get around this by one-offing the CSS, but that's a maintenance issue.  Is there a way I can tell Turbopack to be lenient in its CSS parsing?  The site works fine and gives no such errors when --turbo is not used, asterisk properties and all.

**Comment by masterbater** [comment]
modify your SASS if you are using it

**Reply by jasontaylor137** [reply]
This is a workaround... it's not my css, it's a third-party dependency.  If I change it I can't upgrade it anymore without redoing the change (the maintenance issue I mentioned).

**Reply by Boomika-7** [reply]
Could you let me know if this has been fixed? If it has, could you explain how to fix it?
